build/templates/errors.py.mako

```diff
@@ -55,6 +56,22 @@ class DriverWarning(Warning):
         super(DriverWarning, self).__init__('Warning {0} occurred.\n\n{1}'.format(code, description))
 
 
+% if grpc_supported:
+class RpcError(Error):
```

Comment:
(Short non-PR-specific rant: I'm unhappy with github comments on code versus comments elsewhere, which don't appear to require being resolved and it's not trivial to comment on.)

I think it would be reasonable, when gRPC is supported, to have __init__.py also include this line:
`from nifgen.errors import RpcError # noqa: F401`

Reply:
For better of for worse we don't import specific error classes today. Whether we should or not I don't know but I don't think as part of this PR.

Reply:
I think I have to agree with Marcos. Any customer who needs to distinguish can always import that specifically themselves. So I no longer think it's necessary to auto import it.

Interestingly, I believe the C layer will assign unique new driver error codes and map these gRPC errors in some way. It's not like they have any other option, really. But that doesn't mean we need to be 100% identical there.